src/NServiceBus.Core/NServiceBus.Core.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp70</s:String></wpf:ResourceDictionary>
```

Comment:
If I recall correctly in the past the argument was that we wait to use these new language features until reasonable adoption occurs (was it 6 months for CSharp 6?) to not force outside contributors to immediately upgrade VS // @SimonCropp thoughts?